docs/status.md

```diff
@@ -0,0 +1,58 @@
+# status
+
+```
+cli-utils apply status <dir>
+```
+works for both directories with or without `kustomization.yaml`.
+
+## kustomization directory
+
+First, apply resources from a kustomization directory.
+
+```
+cli-utils apply config/hello
+```
+
+Then, get the status of the applied resources
+```
+cli-utils apply status config/hello
+```
+
+The output is like the following
```

Comment:
what is the exit code if it is done vs not done?

Reply:
The exit code is 0 but the not done case will show the failure.